start_sprint.sh

```diff
@@ -15,22 +15,12 @@ OS=$(uname)
 TIMESTAMP=$(date +"%Y%m%d-%H%M")
 
 #Extract a new ddev D8 core instance to $CWD/sprint-$TIMESTAMP
-mkdir -p sprint-$TIMESTAMP
-tar xpf sprint.tar.xz -C sprint-$TIMESTAMP
+mkdir -p sprint-${TIMESTAMP}
+tar xpf sprint.tar.xz -C sprint-${TIMESTAMP}
 wait
 
 #Update ddev project name
-if [[ "$OS" == "Darwin" ]]; then
-	sed -i '' 's/\[ts\]/'${TIMESTAMP}'/' sprint-$TIMESTAMP/.ddev/config.yaml
-	sed -i '' 's/\[ts\]/'${TIMESTAMP}'/' sprint-$TIMESTAMP/Readme.txt
-	sed -i '' 's/\[ts\]/'${TIMESTAMP}'/' sprint-$TIMESTAMP/start_clean.sh
-	sed -i '' 's/\[ts\]/'${TIMESTAMP}'/' sprint-$TIMESTAMP/start_clean.cmd
-elif [[ "$OS" == "Linux" ]]; then
-	sed -i 's/\[ts\]/'${TIMESTAMP}'/' sprint-$TIMESTAMP/.ddev/config.yaml
-	sed -i 's/\[ts\]/'${TIMESTAMP}'/' sprint-$TIMESTAMP/Readme.txt
-	sed -i 's/\[ts\]/'${TIMESTAMP}'/' sprint-$TIMESTAMP/start_clean.sh
-	sed -i 's/\[ts\]/'${TIMESTAMP}'/' sprint-$TIMESTAMP/start_clean.cmd
-fi
+perl -pi -e "s/\[ts\]/${TIMESTAMP}/g" sprint-${TIMESTAMP}/* sprint-${TIMESTAMP}/.ddev/config.yaml
```

Comment:
It looks like perl is included in git-bash 2.x versions. :+1:

Reply:
```
$ ./start_sprint.sh
Can't do inplace edit: sprint-20180818-1505/drupal8 is not a regular file.
```

Also perl probably is generating some backup files:
```
$ ls
drupal8/  Readme.txt  Readme.txt.bak  start_clean.sh*  start_clean.sh.bak*
```

Both of these did not have any effect on the outcome.